### ISSUE TITLE
sbt installer is unable to download from China

### ISSUE THREAD

**Issue by hope7th** [sbt/sbt#3558]
## steps

download sbt in http://www.scala-sbt.org/download.html

## problem
i even can't download sbt in the Official website，it's suppot Chinese developers so unfriendly...

## notes

sbt version: *insert sbt version*

**Comment by eed3si9n** [comment]
Hi @hope7th could help us debug the problem?
What do you see if you run the following?

```
$ curl -v https://cocl.us/sbt-1.0.1.tgz > sbt-1.0.1.tgz
```

**Comment by eed3si9n** [comment]
Could you also try

```
$ curl -v https://62a44.https.cdn.softlayer.net/sbt/1.0.1/sbt-1.0.1.tgz > sbt-1.0.1.tgz
```

**Comment by OutToLearn** [comment]
I am facing this issue from Pune, India. It's frustrating. I am not able to download the SBT from the official website. After about 25% download completion, it fails due to a "Network error". My internet is working perfectly fine, I've tried different browsers. My friend is facing the same problem from a different machine / ISP in Bangalore. Something is definitely wrong with the website download link.

I have tried downloading the MSI and the ZIP, latest and prior versions. I am setting this up on my Windows 10 system. All links behave in the same erroneous way. Downloads from other websites work perfectly.
